pandas/core/series.py

```diff
@@ -77,12 +73,13 @@ def wrapper(self):
 
     return wrapper
 
+
 # ----------------------------------------------------------------------
 # Series class
 
 
 class Series(base.IndexOpsMixin, strings.StringAccessorMixin,
-             generic.NDFrame,):
+             generic.NDFrame, ):
```

Comment:
I think you have some flake issues. This fine. try to not change lines that don't apply to the patch.